src/System.Management.Automation/utils/tracing/PSEtwLogProvider.cs

```diff
@@ -124,7 +124,17 @@ internal override void LogCommandLifecycleEvent(Func<LogContext> getLogContext,
                     }
                     else
                     {
-                        payload.AppendLine(StringUtil.Format(EtwLoggingStrings.CommandStateChange, logContext.CommandName, newState.ToString()));
+                        if (newState == CommandState.Stopped ||
+                            newState == CommandState.Terminated)
+                        {
+                            // When state is stopped or termianted only log the CommandName
+                            payload.AppendLine(StringUtil.Format(EtwLoggingStrings.CommandStateChange, logContext, newState.ToString()));
```

Comment:
Don't need to explicitly call `ToString()` on formatting arguments here and below.

Reply:
Tnkx